src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -207,7 +171,7 @@ public void AddIssueTracker(TextBufferIssueTracker bufferHandler)
             lock (managers)
             {
                 issueTrackers.Add(bufferHandler);
-                daemon.Ready += bufferHandler.DaemonStarted;
+                //TODO duncanp: daemon.Ready += bufferHandler.DaemonStarted;
```

Comment:
Will be done as part of #926: Delay starting the daemon until a file needs to be analyzed